container/libcontainer/helpers.go

```diff
@@ -90,10 +87,21 @@ func GetStats(cgroupManager cgroups.Manager, networkInterfaces []string) (*info.
 	if len(networkInterfaces) != 0 {
 		// ContainerStats only reports stat for one network device.
 		// TODO(rjnagal): Handle multiple physical network devices.
-		stats.Network, err = sysinfo.GetNetworkStats(networkInterfaces[0])
+		// TODO(rjnagal): Use networking stats directly from libcontainer.
+		net, err := sysinfo.GetNetworkStats(networkInterfaces[0])
 		if err != nil {
 			return stats, err
 		}
+		// Ingress for host veth is from the container.
```

Comment:
I think we need to do this in `container/docker/handler.go`. This function is also used by the raw handler so we'll get flipped stats for host.